app/assets/javascripts/discourse/app/components/d-tooltip.js

```diff
@@ -29,10 +33,12 @@ export default class DiscourseTooltip extends Component {
       this._tippyInstance = tippy(parent, {
         interactive,
         content: element,
-        trigger: this.capabilities.touch ? "click" : "mouseenter",
```

Comment:
We have to be careful here, as I know @awesomerobot and @jordanvidrine use `DTooltip` in many places, including plugins and themes.  I do agree that laptops with touchscreens are tricky, but if we want to fix something for them, the fix probably shouldn't alter the expected behavior on the regular screens. In any case, making tooltips appear on click on desktop isn't related to the thing we're fixing in this PR, and it's worth its own PR and probably discussion, this is controversial.

I'm guessing that made it easier to write a test here? If that's the case, let's just go without a test, I think it's OK in this specific case, because testing this is quite tricky.

One thing we need to do if we go without a test is to mention in the PR description and in the commit message _why_ we're making this change. Something like  "we need to stop event propagation because we want to be able to show `DTooltip` inside on top of other popups". 

Speaking of showing tooltip on click in case we decide to do it in another PR

> Having both does not seem to be an issue.

I'd say this is an issue, this makes the desktop site behave like a site on a touch screen. For example, here I hover on the status, the tooltip appears, then I click it, and after my click the tooltip never disappear: 

https://user-images.githubusercontent.com/1274517/231153205-df602808-7a6d-4d34-8a91-63d35256a32f.mov

This is unusual behavior for desktop and feels like a bug.

Another example, on the assign modal click leads to showing tooltip:

https://user-images.githubusercontent.com/1274517/231184249-0953793a-fde6-462d-8ef0-66611f191a82.mov


While I would expect click to lead to choosing the row, like this:

https://user-images.githubusercontent.com/1274517/231184137-6dee1356-734f-485a-bda9-4b4a8885254d.mov